runtime/lua/vim/lsp/health.lua

```diff
@@ -0,0 +1,25 @@
+local M = {}
+
+--- Performs a healthcheck for LSP
+function M.check_health()
+  local report_info = vim.fn['health#report_info']
+  local report_warn = vim.fn['health#report_warn']
+
+  local log = require('vim.lsp.log')
+  local current_log_level = log.get_level()
+  report_info(string.format("Current log level : %s", log.levels[current_log_level]))
+
+  if current_log_level < log.levels.WARN then
+    report_warn(string.format("Using non-default log level will result in poor runtime performance and increased disk usage"))
```

Comment:
I wanted to emphasize that by default we don't use a log level that would cause an issue with diskspace, and that users should remove the line from their config. I can change this.